config/manifests/bases/watcher-operator.clusterserviceversion.yaml

```diff
@@ -12,7 +12,6 @@ metadata:
     features.operators.openshift.io/token-auth-azure: "false"
     features.operators.openshift.io/token-auth-gcp: "false"
     operatorframework.io/suggested-namespace: openstack
```

Comment:
should be `openstack-operators` ?

Reply:
isnt this the suggested namespace for resouce created
that are reconsiled by the operator?

or is this for the operator itself?

we use OpenStack in nova https://github.com/openstack-k8s-operators/nova-operator/blob/main/config/manifests/bases/nova-operator.clusterserviceversion.yaml#L14
and keystone
https://github.com/openstack-k8s-operators/keystone-operator/blob/main/config/manifests/bases/keystone-operator.clusterserviceversion.yaml#L9

Reply:
oh I think you are correct, sorry for that.